examples/TestDriver/ios/TestDriver/AppDelegate.m

```diff
@@ -38,6 +38,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"TestDriver"
                                                initialProperties:nil
                                                    launchOptions:launchOptions];
+  [rootView setValue:@true forKey:@"heapIgnore"];
```

Comment:
Mind adding a comment about why you're doing this (I'm guessing you're doing it to suppress iOS native events, but may be worth calling that out to the next reader)

Reply:
Done